addons/stock_account/tests/test_stockvaluationlayer.py

```diff
@@ -1076,3 +1076,43 @@ def test_avco_and_credit_note(self):
         self.assertEqual(len(anglo_lines), 2)
         self.assertEqual(abs(anglo_lines[0].balance), 10)
         self.assertEqual(abs(anglo_lines[1].balance), 10)
+
+    def test_dropship_return_accounts(self):
```

Comment:
Could you add a second test where you return the dropship in WH/Stock instead of vendor ? The return should create 

Account | Debit | Credit
-- | -- | -- 
Stock Interim (Delivered)  | $ 0.00 | price |  
Stock Valuation  | price | $ 0.00 |  
Thank you

Reply:
Done! :slightly_smiling_face: